File: Programmers/Lv.1/문자열 내 p와 y의 개수/12916.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;

        int pCnt = 0, yCnt = 0;
        for (String ss : s.split("")) {
            if ("P".equals(ss.toUpperCase())) {
                pCnt++;
            } else if ("Y".equals(ss.toUpperCase())) {
                yCnt++;
            }
        }
        
        answer = (pCnt == yCnt);

        return answer;
    }
}

File: Programmers/Lv.1/문자열 내 p와 y의 개수/12916.js
function solution(s){
    let answer = true;
    
    let pCnt = 0, yCnt = 0;
    s.split('').forEach(v => {
       if (v.toUpperCase() === 'P') {
           pCnt++;
       } else if (v.toUpperCase() === 'Y') {
           yCnt++;
       }
    });
    
    answer = (pCnt === yCnt);

    return answer;
}
